corehq/apps/reports/views.py

```diff
@@ -2120,3 +2123,131 @@ def project_health_user_details(request, domain, user_id):
         'groups': ', '.join(g.name for g in Group.by_user_id(user_id)),
         'submission_by_form_link': submission_by_form_link,
     })
+
+
+class TableauServerView(BaseProjectReportSectionView):
```

Comment:
As long as the feature flag exists, these views should check it. They should also check user permissions. I think these views should check `edit_motech`, to be similar to the `BiometricIntegrationView`, another integrations-type view, but product should confirm that. It looks like the current integration views are a bit inconsistent. It's possible we should have stricter permissions for the USH integrations like `GaenOtpServerSettingsView`, which are currently open to all users.

The final flag & permissions logic should also be applied in `tabclasses.py`.

Reply:
Thank you for flagging this! Confirmed that the feature flag **_is_** needed for now, but still awaiting the user permissions answer.

Reply:
_Update_: having these setting under a feature flag with no other user permissions checks is sufficient for the foreseeable future